corehq/sql_accessors/migrations/0055_set_form_modified_on.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.7 on 2017-07-06 21:18
+from __future__ import unicode_literals
+
+from __future__ import absolute_import
+from django.db import migrations
+
+from corehq.form_processor.models import XFormInstanceSQL, XFormOperationSQL
+from corehq.sql_db.operations import RawSQLMigration
+
+migrator = RawSQLMigration(('corehq', 'sql_accessors', 'sql_templates'), {
+    'FORM_STATE_DELETED': XFormInstanceSQL.DELETED,
+    'FORM_STATE_ARCHIVED': XFormInstanceSQL.ARCHIVED,
+    'FORM_STATE_NORMAL': XFormInstanceSQL.NORMAL,
+    'FORM_OPERATION_ARCHIVE': XFormOperationSQL.ARCHIVE,
+    'FORM_OPERATION_UNARCHIVE': XFormOperationSQL.UNARCHIVE,
+})
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('sql_accessors', '0054_drop_reindexa_accessor_functions'),
```

Comment:
I think we'll need to figure out something with https://github.com/dimagi/commcare-hq/pull/18538 since it'll have a conflicting migration. Was that successfully deployed to staging?

Reply:
> Was that successfully deployed to staging?

it was not. I can reconcile migrations before I do that.